fpdf/image_parsing.py

```diff
@@ -30,6 +31,8 @@ def load_image(filename):
     # if a bytesio instance is passed in, use it as is.
     if isinstance(filename, BytesIO):
         return filename
+    if isinstance(filename, Path):
```

Comment:
```suggestion
    elif isinstance(filename, Path):
```
👍

Reply:
I have to revert it back to what it was to make pylint happy, sorry